pan-domain-auth-verification/src/test/scala/com/gu/pandomainauth/PublicSettingsTest.scala

```diff
@@ -1,38 +1,11 @@
 package com.gu.pandomainauth
 
-import com.gu.pandomainauth.service.TestKeys.testPublicKey
-import com.gu.pandomainauth.service.{Crypto, TestKeys}
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.EitherValues
+import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
 
-class PublicSettingsTest extends AnyFreeSpec with Matchers with EitherValues with ScalaFutures {
-  "validateKey" - {
```

Comment:
test cases have moved to `CryptoConfTest` to reflect the refactor in the code above 👍